src/ui/components/Widgets/Chart/hooks/useLoadingChart.ts

```diff
@@ -496,6 +508,34 @@ export const useLoadingChart = (props: LoadingChartHookProps) => {
         }
     }, [throttledInViewportCheck]);
 
+    /**
+     * reload chart by timer when the _autoupdate param is passed
+     */
+    const reloadChart = React.useCallback(() => {
+        const autoupdateIntervalMs = Number(autoupdateInterval) * 1000;
+        if (autoupdateIntervalMs) {
+            const timeSinceLastReload = new Date().getTime() - (lastReloadAt || 0);
+            const reloadIntervalRemains = autoupdateIntervalMs - timeSinceLastReload;
+
+            if (
+                (!isPageHiddenCurrent || isEmbeddedMode()) &&
+                reloadIntervalRemains <= 0 &&
+                reloadTimeout
+            ) {
+                lastReloadAt = new Date().getTime();
+                loadChartData();
+            }
+
+            reloadTimeout = setTimeout(() => reloadChart(), autoupdateIntervalMs);
+        }
+    }, [loadChartData, autoupdateInterval]);
```

Comment:
I suppose you need to add `isPageHiddenCurrent` in deps, perhaps `reloadChart` too.

Reply:
Now it's part of the ref